src/satellite_images_nso/_manipulation/nso_manipulator.py

```diff
@@ -255,19 +260,15 @@ def generate_vegetation_height_channel(
     return channel
 
 
-def transform_vector_to_pixel_df(path_to_vector, add_ndvi_column=False):
+def transform_vector_to_pixel_df(path_to_vector):
     """
     Maps a rasterio satellite vector object to a geo pandas dataframe per pixel.
-    With the corresponding x and y coordinates and NVDI.
-
+    TODO: This method is not used anymore, could be deleted.

```

Comment:
I would either delete it immediately or remove the comment and not delete it. 

Or as Yoda would say:
https://tenor.com/wSwQ.gif

Reply:
this code indeed is already covered in the data loader and thus will be deleted